preseed.cfg

```diff
@@ -2,11 +2,15 @@
 # kernel is to be installed.
 d-i base-installer/kernel/image string linux-image-novena
 
+# This command is run immediately before the partitioner starts. It may be
+# useful to apply dynamic partitioner preseeding that depends on the state
+# of the disks (which may not be visible when preseed/early_command runs).
+d-i preseed/early_command string mkdir /sd ; mount /dev/mmcblk0p1 /sd ; mv /sd/uInitrd /sd/uInitrd-install ; sync ; umount /sd ; sync ; rmdir /sd
```

Comment:
That would be a mistake on my part. Honestly, we can probably delete xorg.conf and get rid of that chuck in preseed.cfg (my Novena works just fine without xorg.conf).